desc/compute/_basis_vectors.py

```diff
@@ -3223,17 +3223,61 @@ def _e_sub_zeta_zz(params, transforms, profiles, data, **kwargs):
     transforms={},
     profiles=[],
     coordinates="rtz",
-    data=["e^rho", "e^theta", "e^zeta", "alpha_r", "alpha_t", "alpha_z"],
+    data=["periodic(grad(alpha))", "secular(grad(alpha))"],
 )
 def _grad_alpha(params, transforms, profiles, data, **kwargs):
-    data["grad(alpha)"] = (
-        data["alpha_r"] * data["e^rho"].T
+    data["grad(alpha)"] = data["periodic(grad(alpha))"] + data["secular(grad(alpha))"]
+    return data
+
+
+@register_compute_fun(
+    name="periodic(grad(alpha))",
```

Comment:
Does using periodic grad(alpha) and gbdrift fix the convergence issue in Gamma_C? When you increase the field line length does Gamma_C-> 0 or a fixed values?

Reply:
I haven't checked whether we can avoid the additional accuracy demanded by the integrals so that the secular term in Gamma_c Velasco vanishes by just removing the secular term from the integrals, essentially as Nemov has done.